hal/src/main/native/athena/Notifier.cpp

```diff
@@ -161,8 +168,14 @@ HAL_NotifierHandle HAL_InitializeNotifier(int32_t* status) {
 
 HAL_Bool HAL_SetNotifierThreadPriority(HAL_Bool realTime, int32_t priority,
```

Comment:
Should this function and the priority setting in HAL_InitializeNotifier() be protected by a mutex? If I recall correctly, we want the HAL to be thread-safe.

Reply:
We're already holding a mutex in initialize notifier, so we could just lock that mutex in the priority setting func